notebooks/04-Creating_ASDF_Files.ipynb

```diff
@@ -0,0 +1,753 @@
+{
```

Comment:
<p>Should we standardize what the variable name is for the <code>AsdfFile</code> object? For example I use <code>af</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>